libs/blocks/twitter/twitter.js

```diff
@@ -3,7 +3,8 @@ import { createIntersectionObserver, createTag, loadScript, isInTextNode } from
 export default function init(a) {
   if (isInTextNode(a)) return;
   const embedTwitter = () => {
-    const anchor = createTag('a', { href: a.href });
+    const url = a.href.replace('https://x.com', 'https://twitter.com');
```

Comment:
Why does it need to be converted to twitter.com?

Reply:
I added this as x.com urls are not getting embedded. I see some discussion around this few forums and looks it is not supported yet as twitter is redirecting x.com url tweets to twiteer.com as of now.
Here is x.com link where you can see it redirects to twitter.com https://x.com/Adobe/status/1714718693161312496?s=20

Reply:
https://publish.twitter.com/ - the embedded post is twitter.com